src/templates/globals/medallia/index.tsx

```diff
@@ -0,0 +1,41 @@
+import Head from 'next/head'
+import { useEffect } from 'react'
+import {
+  getSurveyNumber,
+  loadForm,
+  onMedalliaLoaded,
+  setWindowVaSurvey,
+} from '@/lib/utils/medallia'
+
+export function MedalliaAssets() {
+  const scriptId = process.env.BUILD_TYPE === 'production' ? 2 : 5
+
+  useEffect(() => {
+    if (process.env.NEXT_PUBLIC_BUILD_TYPE === 'localhost') {
+      onMedalliaLoaded(() => {
+        const surveyNumber = getSurveyNumber(window.location.pathname, false)
+        const neb_status = loadForm(surveyNumber)
+        if (neb_status === true) {
+          /*eslint-disable-next-line*/
+          console.log(`survey number ${surveyNumber} has loaded`)
+        }
+      })
+    }
+
+    const isVamcPage = window.location.pathname.includes('health-care')
+    if (isVamcPage) {
+      setWindowVaSurvey('mcenter')
+    }
+  }, [])
+
+  return (
+    <Head>
+      <script
```

Comment:
This is _definitely_ something that I need to fix. I had a lot of different hurdles while working on this and I didn't want to get bogged down with one while working on another. I had a mental note to come back to the `<script>` tags to utilize Next's `<Script>` tags, but I never got to it and then lost track of it. Thanks for catching this (I'm not sure why I didn't see the warning, but probably because there were a lot of moving pieces as I tried to get it across the line). I'll fix asap.

Reply:
This is now fixed.